examples/footer/index.html

```diff
@@ -11,52 +11,52 @@
     <footer>
       <div class="container">
         <div class="row">
-          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3">
             <a class="navbar-brand" href="https://stjude.cloud">
               <span class="logo"></span>
             </a>
             <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
           </div>
           <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-about">
-            <h6>About St. Jude</h6>
+            <h6>About</h6>
             <ul>
               <li>
-                <a href="https://www.stjude.org">Home</a>
+                <a href="https://stjude.cloud" title="St. Jude Cloud Home">Home</a>
               </li>
               <li>
-                <a href="https://www.stjude.org/about-st-jude.html">About</a>
+                <a href="https://stjude.cloud/privacy-policy" title="Privacy Policy">Privacy Policy</a>
               </li>
               <li>
-                <a href="https://stjude.org/contact-us.html">Contact</a>
+                <a href="https://stjude.cloud/terms-of-use" title="Contact">Contact</a>
```

Comment:
I like that we are linking to SJC, but this link and anchor seem incongruent (also below).